operator/internal/validation/openshift/otlp.go

```diff
@@ -0,0 +1,96 @@
+package openshift
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	lokiv1 "github.com/grafana/loki/operator/api/loki/v1"
+)
+
+var (
+	requiredStreamLabels = []string{
+		"k8s.namespace.name",
+		"kubernetes.namespace_name",
+		"log_source",
+		"log_type",
+		"openshift.cluster.uid",
+		"openshift.log.source",
+		"openshift.log.type",
+	}
+
+	optionalStreamLabels = []string{
+		"k8s.container.name",
+		"k8s.cronjob.name",
+		"k8s.daemonset.name",
+		"k8s.deployment.name",
+		"k8s.job.name",
+		"k8s.node.name",
+		"k8s.pod.name",
+		"k8s.statefulset.name",
+		"kubernetes.container_name",
+		"kubernetes.host",
+		"kubernetes.pod_name",
+		"service.name",
+	}
```

Comment:
These are replicated with what we have in `internal/manifests/openshift/otlp.go` shouldn't we try and merge both?

Reply:
Ah right, this was one of the remaining points. The same set also appears in the other package, but we can not use it from here because that would cause an import loop. Need to find a place to move this to...